feature_engine/pipeline/pipeline.py

```diff
@@ -442,3 +442,51 @@ def fit_predict(self, X, y=None, **params):
                 Xt, yt, **params_last_step.get("fit_predict", {})
             )
         return y_pred
+
+
+def make_pipeline(*steps, memory=None, verbose=False):
+    """Construct a :class:`Pipeline` from the given estimators.
+
+    This is a shorthand for the :class:`Pipeline` constructor; it does not
+    require, and does not permit, naming the estimators. Instead, their names
+    will be set to the lowercase of their types automatically.
```

Comment:
Just to know:
Can we have other types outside string that can be set to lowercase?